docs/named_releases.rst

```diff
@@ -0,0 +1,344 @@
+#######################
+Open edX Named Releases
+#######################
+
+The Open edX community can share knowledge and improvements more easily when most people use the same stable, consistent
+version of the Open edX codebase. To that end, edX is creating a series of "Open edX named releases", which are distinct
+from the daily deployments to edx.org and have a longer release cycle (on the order of several months between each
+release). These releases will be tested both by edX and by the Open edX community, and with each subsequent release we
+will include instructions for how to upgrade from the previous Open edX named release.
+
+Latest Open edX Release
+-----------------------
+The latest supported release line is Hawthorn_, based on code from 2018-07-03.
+
+All Open edX Releases
+---------------------
+
+.. contents::
+   :local:
+   :depth: 1
+
+Note that the latest release (Hawthorn) is the only supported release. We currently do not have the ability to support
+more than one release at a time.
+
+For documentation on the latest release, please visit http://edx.readthedocs.org/projects/open-edx-release-notes/en/latest/
+
+Every release line (Dogwood, Eucalyptus, etc) has a branch that accumulates changes destined for that release.  When it
+is time to make a new release in the line, we tag that branch.  When a release line is no longer supported, we might
+still add important fixes to the branch, but not make a new tagged release.  At that point, someone installing that line
+will want to install from the branch, not the tags.
+
+A note about Vagrant Box Files:
+
+* Releases before Hawthorn had Vagrant box files. Hawthorn and beyond do not.
+* Not every release needed new box files.
+* Box files the same as the previous release are marked with an asterisk \*.
+* Hashes are SHA1 hashes of the box file, not git commit hashes.
+
+Hawthorn
+~~~~~~~~
+
+* **Code cut date:** 2018-07-03
+* **Status:** supported
+* **Note:** Devstack is now based on Docker.
+* `Release Notes <https://edx.readthedocs.io/projects/open-edx-release-notes/en/latest/hawthorn.html>`_
+
+.. list-table::
+   :header-rows: 1
+
+   * - Release Name
+     - Release Date
+     - Git Tag
+
+   * - Hawthorn.2
+     - 2018-09-26
+     - open-release/hawthorn.2
+
+   * - Hawthorn.1
+     - 2018-08-07
+     - open-release/hawthorn.1
+
+Ginkgo
+~~~~~~~~
+
+* **Code cut date:** 2017-07-06
+* **Status:** unsupported
+* **Latest:** open-release/ginkgo.master
+* `Release Notes <https://edx.readthedocs.io/projects/open-edx-release-notes/en/latest/ginkgo.html>`__
+
+.. list-table::
+   :header-rows: 1
+
+   * - Release Name
+     - Release Date
+     - Git Tag
+     - Vagrant Box Files
+
+   * - GInkgo.2
```

Comment:
G_I_ => G_i_

Reply:
This is how it was in the original doc - as I said, I didn't make any editorial changes - just reformatted.